bin/validate_swift_package

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+#
+# Any arguments passed to this script will be passed through to `fastlane`.
+# If no argument is passed, the `test` lane will be called by default.
+
+if [[ ! -f Package.swift ]]; then
+    echo "This repo is not a Swift package."
+    exit
```

Comment:
Have you considered using `exit 1`?

IMHO, if we call this command on a repo that's not setup for it, it would be good to fail the build because clearly there's a misalignment in the setup.

Reply:
👍 Followed up in https://github.com/Automattic/bash-cache-buildkite-plugin/pull/35